### ISSUE TITLE
JAWS And Chrome RadioGroups using aria-activedescendant, Number and Position of Radio Buttons Are Not reported when Virtual PC Cursor Is Active

### ISSUE THREAD

**Issue by jhartleytpgi** [FreedomScientific/VFO-standards-support#852]
### Summary
When navigating through a group of radio buttons with the virtual cursor active, neither the position of the radio button or the number of radio buttons in the group are Announced.
The above is true when navigating through a radioGroup using the following JAWS commands:

- f and Shift+f -- Focus to next field -- FocusToPriorField
- a and Shift+a -- MoveToNextRadioButton -- MoveToPriorRadioButton
- Tab and Shift+Tab (When VPC is Active)
- Down Arrow  and Up Arrow (When VPC is active)
- JAWSKEY+TAB  -- SayWindowPromptAndText (When VPC Is Active).

### Expected result
JAWS should convey the position and number of the radio buttons in the group while navigating. For example, if there was a group of three radio buttons with the first one being labeled "regular crust",  and the user navigated to the first radio button, JAWS should announce "Regular crust radio button checked 1 of 3".

### Actual result
If the user navigates to the first radio button in the group of three (Regular Crust), JAWS announces "Regular crust radio button checked", but does not announce the position of the radio button (1) or the number of radio buttons in the group (3).

### Example
https://aria-at.netlify.app/tests/radiogroup-aria-activedescendant/reference/2022-4-7_113015/radio-activedescendant.setfocusonfirstradiobutton

#### JAWS version and build number
2024.2406.121

 
#### Operating System and version
Windows 10 / Windows 11

#### Browser and version:
Google Chrome 126.0.6478.183 (Official Build) (64-bit)
